flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -169,4 +186,113 @@ public ExecutionConfig disableObjectReuse() {
 	public boolean isObjectReuseEnabled() {
 		return objectReuse;
 	}
+
+	// --------------------------------------------------------------------------------------------
+	//  Registry for types and serializers
+	// --------------------------------------------------------------------------------------------
+
+	/**
+	 * Registers the given Serializer as a default serializer for the given type at the
```

Comment:
If you want I can also change it. I've another change for the Kryo system pending, so I can do the change in the course of that.